ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -622,7 +622,7 @@ objects:
         trap 'teardown' EXIT
         trap 'jobs -p | xargs -r kill || true; exit 0' TERM
 
-        for i in $(seq 1 220); do
+        for i in $(seq 1 140); do
```

Comment:
What is this change for?

Reply:
Found it, from the commit message: "Timeout of test is reduced from 3.6 hours to 2.3 hours."

Reply:
This loop triggers the `teardown` trap (see line 622) if `/tmp/shared/exit` exists, else after 140 minutes (see `sleep` line 629), instead of 220 minutes. I believe that this represents the longest that Prow will let a job run.

The rationale I can imagine here is: since the new infrastructure is faster, and incidentally we need to increase efficiency, we limit the max lenght of the jobs to reduce overlapping between jobs.

We might try and see what the outcome is: if now may jobs time out, then we can revert

Reply:
Decreasing the job timeout makes sense to me, however I'd be cautious since this change is global and affects jobs running on both MOC and Vexxhost.
Also, only taking Vexxhost into consideration, the serial jobs frequently cross the 2.3 hours mark, the latest successful serial job was 2.5 hours for instance. I think the new limit is a bit too aggressive.

Reply:
kk let me revert that then, we'll deal with thi sissue separately